src/data.py

```diff
@@ -280,6 +283,19 @@ def get_trillium_menu():
   statics_json = requests.get(STATIC_MENUS_URL).json()
   return parse_dining_items(statics_json['Trillium'][0])
 
+def fill_empty_menus(eateries):
+  for eatery in eateries.values():
+    for operating_hour in eatery.operating_hours:
+      # need to choose event with data we want
+      if len(operating_hour.events) <= 1 or len(operating_hour.events[0].menu) == 0:  # ignore these
```

Comment:
why is this one `<= 1 `and the one with stations `== 0`

Reply:
the one on line 131 with ==1 is to prevent us from making an empty FoodStaionType object.  When we pull from the cornell dining json, if there is no menu, they return by defualt a menu of one station with empty items.  This adds unnecessary information so we are stopping the code from making a worthless object and instead just return an empty list for the menu.  This also makes it easier in the later part to determine if we need to copy a menu.

Reply:
This one with <=1 is there to skip over operating hours with only one or less events because there will be no reason for us to copy information into a later event.